src/go/MONIT/alert.go

```diff
@@ -0,0 +1,658 @@
+
+//boolean for detailed output for an alert
+var details *bool
+
+//Sort Label
+var sortLabel string
+
+//verbose defines verbosity level
+var verbose int
+
+//-------VARIABLES-------
+
+//-------MAPS-------
+
+//Map for storing filtered alerts
+var filteredAlerts map[string]int
+
+//Map for severityLevel (Sorting Helper)
+var severityLevel map[string]int
+
+//Map for storing Alert details against their name
+var alertDetails map[string]interface{}
+
+//-------MAPS-------
+
+//AlertManager API acceptable JSON Data for GGUS Data
+type amGGUSJSON struct {
+	Labels struct {
+		Alertname string `json:"alertname"`
+		Severity  string `json:"severity"`
+		Service   string `json:"service"`
+		Tag       string `json:"tag"`
+		Priority  string `json:"Priority"`
+		Scope     string `json:"Scope"`
+		Site      string `json:"Site"`
+		VO        string `json:"VO"`
+		Type      string `json:"Type"`
+	} `json:"labels"`
+	Annotations struct {
+		Priority        string `json:"Priority"`
+		ResponsibleUnit string `json:"ResponsibleUnit"`
+		Scope           string `json:"Scope"`
+		Site            string `json:"Site"`
+		Status          string `json:"Status"`
+		Subject         string `json:"Subject"`
+		TicketID        string `json:"TicketID"`
+		Type            string `json:"Type"`
+		VO              string `json:"VO"`
+		URL             string `json:"URL"`
+	} `json:"annotations"`
+	StartsAt time.Time `json:"startsAt"`
+	EndsAt   time.Time `json:"endsAt"`
+}
+
+//AlertManager GET API acceptable JSON Data struct for GGUS data
+type ggusData struct {
+	Data []amGGUSJSON
+}
+
+//AlertManager API acceptable JSON Data for CERN SSB Data
+type amSSBJSON struct {
+	Labels struct {
+		Alertname   string `json:"alertname"`
+		Severity    string `json:"severity"`
+		Service     string `json:"service"`
+		Tag         string `json:"tag"`
+		Type        string `json:"type"`
+		Description string `json:"description"`
+		FeName      string `json:"feName"`
+		SeName      string `json:"seName"`
+	} `json:"labels"`
+	Annotations struct {
+		Date             string    `json:"date"`
+		Description      string    `json:"description"`
+		FeName           string    `json:"feName"`
+		MonitState       string    `json:"monitState"`
+		MonitState1      string    `json:"monitState1"`
+		SeName           string    `json:"seName"`
+		ShortDescription string    `json:"shortDescription"`
+		SsbNumber        string    `json:"ssbNumber"`
+		SysCreatedBy     string    `json:"sysCreatedBy"`
+		SysModCount      string    `json:"sysModCount"`
+		SysUpdatedBy     string    `json:"sysUpdatedBy"`
+		Type             string    `json:"type"`
+		UpdateTimestamp  time.Time `json:"updateTimestamp"`
+	} `json:"annotations"`
+	StartsAt time.Time `json:"startsAt"`
+	EndsAt   time.Time `json:"endsAt"`
+}
+
+//AlertManager GET API acceptable JSON Data struct for SSB data
+type ssbData struct {
+	Data []amSSBJSON
+}
+
+//AlertManager API acceptable JSON Data for Grafana Alerts
+type amGrafanaJSON struct {
```

Comment:
replace with amJSON struct I provided in review

Reply:
Fixed this issue.